usagov/usagov-2021
# USAGOV-1948-wizard-expand-all: Add an expand/collapse all button to the wizard overview form.

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1948

## Description

Create wizard expand/collapse js file, declare it in a new libraries.yml, attach it to the wizard overview form with a button.

## Type of Changes

- [ X ] New Feature
  - Add screenshot showing what it should look like

## Testing Instructions


1. Navigate to the wizard overview form, /admin/structure/taxonomy/manage/wizard/overview.
2. The form loads as expected with all accordions closed.
3. Click the "Expand/Collapse" button.
4. The button toggles the state of the accordions, either opening them all or closing them all as expected.

### Change Requirements



## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
5. Find the commit of this pull request.
6. Build and deploy the changes.
7. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
8. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 7. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8. 8.
